weatherapplication.client/src/index.html

```diff
@@ -1,177 +1,297 @@
-        }
-      }, timeout())
-    };
+              if (document.readyState == "complete") {
+                // Alpha
+                alpha();
+
+                // Test
+                test();
+              }
+            });
+
+          // Insertion
+          document.body.appendChild(data);
+
+          // Style > <body> > Overflow
+          document.body.style = (
+            "overflow: hidden !important; pointer-events: none !important; user-drag: none !important; user-select: none !important;" +
+            (document.body.getAttribute("style") || " ")
+          ).trim();
+
+          // Modification > Data
+          // Inner HTML
+          data.innerHTML =
+            "<style media=all type=text/css>" +
+            "body::selection {" +
+            "background-color: transparent !important;" +
+            "text-shadow: none !important" +
+            "} " +
+            "@keyframes rotate {" +
+            "0% { transform: rotate(0) }" +
+            "to { transform: rotate(360deg) }" +
+            "}" +
+            "</style>" +
+            "<div style='animation: rotate 1s ease-in-out 0s infinite backwards; border: " +
+            loaderScreenWidth +
+            "px solid rgba(0, 0, 0, " +
+            loaderScreenOpacity +
+            "); border-top-color: rgba(0, 51, 255, " +
+            loaderScreenOpacity +
+            "); border-radius: 50%; height: 75px; margin: 0 auto; margin-bottom: " +
+            loaderScreenMargin +
+            "px; width: 75px'> </div>" +
+            '<small style=\'color: rgba(127, 127, 127, .675); font-family: "Open Sans", "Calibri Light", Calibri, sans-serif; font-size: ' +
+            loaderScreenFontSize +
+            "px !important; margin: 0 auto; margin-top: " +
+            loaderScreenMargin +
+            "px; text-align: center'> " +
+            loaderScreenMessage +
+            " </small>";
+
+          // Style
+          data.style =
+            "align-items: center; background-color: rgba(255, 255, 255, .98); display: flex; flex-direction: column; height: " +
+            innerHeight +
+            "px; justify-content: center; left: 0; margin: auto; max-height: 100% !important; max-width: 100% !important; min-height: 100vh; min-width: 100vh; position: fixed; top: 0; transition: " +
+            loaderScreenTransition +
+            "s ease-in-out; width: " +
+            innerWidth +
+            "px; z-index: 2147483647";
 
-    /* Logic
+          // Function
+          // Alpha
+          function alpha() {
+            // Clear Interval
+            clearInterval(metadata);
+          }
+
+          // Test
+          function test() {
+            // Style > Data
+            // Background Color
+            data.style.backgroundColor = "transparent";
+
+            // Opacity
+            data.style.opacity = 0;
+
```

Comment:
The entire loader that we see when we move between pages should be moved to a separate component called "load-spinner" so we can give it time to exist. For example, we want to see "load-spinner" exactly 3 seconds. For this functionality, we don't need remove it from HTML manually, we need simple *ngIf and few variables